sematic/ui/src/pipelines/RunDetailsPanel.tsx

```diff
@@ -74,17 +82,15 @@ export function RunDetailsPanel() {
                 <Typography variant="h6">{selectedRun.name}</Typography>
                 <Typography fontSize="small" color="GrayText" component="span">
                   <code style={{ fontSize: 12 }}>
-                  ID: {selectedRun.id}
-                  {
-                    selectedRun.original_run_id &&
-                    /*
-                      TODO #278: replace full id with a 6-character trimmed
-                      link using "./Notes/RunId"
-                    */
-                    <> (cloned from {selectedRun.original_run_id})</>
-                  }
+                  ID: {selectedRun.id} <CopyButton text={selectedRun.id} message="Copied run ID" />
                   </code>
                 </Typography>
+                {
+                    selectedRun.original_run_id &&
+                    <StyledText>
+                      cloned from <RunId runId={selectedRun.original_run_id} copy={false} />
```

Comment:
Can we make sure only the IDs are monospace? the parenthesis and "cloned from" should be the regular font.

Reply:
Sure, how close is this to what you want?

<img width="820" alt="image" src="https://user-images.githubusercontent.com/1046489/213529135-28f9c8ff-0e20-4f00-b943-5746e429a337.png">

Reply:
Can we make it the same grey color? Thank you. Same for the copy icon.

Reply:
Sure, how about this?

<img width="793" alt="image" src="https://user-images.githubusercontent.com/1046489/213768191-99eb024b-0d14-438b-bdcd-6703fc4e1d7d.png">